sql/core/src/test/resources/sql-tests/inputs/postgreSQL/interval.sql

```diff
@@ -342,3 +333,11 @@ SELECT interval '1 2:03:04' minute to second;
 -- select make_interval(secs := 'inf');
 -- select make_interval(secs := 'NaN');
 -- select make_interval(secs := 7e12);
+
+-- test justify_hours() and justify_days()
+-- [SPARK-29390] Add the justify_days(), justify_hours() and justify_interval() functions
```

Comment:
do we still need this?

Reply:
I have some similar PRs which also modify PostgreSQL ported tests, mostly will be requested to leave or enable these tests. I guess they just want to track the progress of PostgreSQL feature parity in Spark.

Reply:
I don't get it. This is a TODO and now it's done. Why still keep it?

Reply:
I'd rather remove these too. I will push a new commit to delete these.